sentry-actix/src/lib.rs

```diff
@@ -180,7 +185,7 @@ impl Default for Sentry {
 
 impl<S, B> Transform<S, ServiceRequest> for Sentry
 where
-    S: Service<ServiceRequest, Response = ServiceResponse<B>, Error = Error>,
+    S: Service<ServiceRequest, Response = ServiceResponse<B>, Error = Error> + 'static,
```

Comment:
Why exactly is the `'static` bound and the `Rc` needed?
If it is indeed unavoidable, better to use `Arc`, as we are talking about (potentially multithreaded) servers here.

Reply:
I needed the RC + 'static because otherwhise i was getting lifetime issues. Wasn't 100% sure if i should use `Rc` or `Arc` after u asked so i ask on the Actix discord and the lead of Actix confirm that `Rc` was the right choice.
source: https://discord.com/channels/771444961383153695/1337163852118491136